docs_src/microservices/application/V3Migration.md

```diff
@@ -6,17 +6,17 @@ title: App Services - V3 Migration Guide
 
 ## Configuration
 
-The migration of any Application Service's configuration starts with migrating configuration common to all EdgeX services. See the [V3 Migration of Common Configuration](../../configuration/V3MigrationCommonConfig) section for details including the change from **TOML** format to **YAML** format for the configuration file. The remainder of this section focuses on configuration specific to Application Services.
+The migration of any Application Service's configuration starts with migrating configuration common to all EdgeX services. See the [V3 Migration of Common Configuration](../configuration/V3MigrationCommonConfig.md) section for details including the change from **TOML** format to **YAML** format for the configuration file. The remainder of this section focuses on configuration specific to Application Services.
```

Comment:
Add a note here about URI for files?

Reply:
What are you thinking should be added?

Reply:
maybe something like: The new YAML configuration file can either be local to the file system or remote accessible via URI.

Reply:
Ok, as a new in 3.1 edgey